src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -5,4 +5,23 @@
     position: absolute;
     bottom: 10px;
     padding: 15px;
+}
+
+/*Allows map to display*/
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+/*Display the map in the center of the iframe, 
```

Comment:
nit.
For multiline comments, leave the starting and ending lines blank besides the comment strings, and end it with a period

```
/*
 * Comment can go here and can,
 * span multiple lines.
 */
```

Reply:
fixed